client/my-sites/plugins/education-footer/index.tsx

```diff
@@ -210,7 +210,7 @@ function ReadMoreLink() {
 	const { __ } = useI18n();
 
 	return (
-		<CardText color="var(--studio-blue-50)">
+		<CardText color="#3858e9">
```

Comment:
Is there somewhere we can put these as variables?

Reply:
Hmmm i can look into this , possibly follow up PR